dpbench/configs/framework_info/numba_cuda.toml

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[framework]
+simple_name = "numba"
```

Comment:
it conflicts with numba

Reply:
I have made the changes to `framework.py` so that this change works. Also, verified that numba_cuda implementations run successfully.